config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml

```diff
@@ -392,9 +392,9 @@ periodics:
           - name: API_SERVER_FEATURE_GATES
             value: "InPlacePodVerticalScaling=true"
           - name: GINKGO_FOCUS
-            value: (\[Feature:InPlacePodVerticalScaling\]|\[Feature:NodeLogQuery\]|\[sig-windows\]) # run the features and a few windows related tests
+            value: (\[Feature:InPlacePodVerticalScaling\]|\[Feature:NodeLogQuery\]|\[sig-windows\]|should.be.able.to.gracefully.shutdown.pods.with.various.grace.periods) # run the features and a few windows related tests
           - name: NODE_FEATURE_GATES
-            value: "InPlacePodVerticalScaling=true,NodeLogQuery=true,PodAndContainerStatsFromCRI=true,RuntimeClassInImageCriApi=true"
```

Comment:
Right now this test pass runs the tests in parallel, do we need to run the tests in serial if we are adding shutdown tests?

Can you add to the labels for this one?
>    preset-capz-serial-slow: "true"